### PR TITLE
Turn on / off heating for all sky camera

### PR DESCRIPTION
**Pull request by sfoale**

### CODE REVIEW

**Review by markBowman — COMMENTED** [review]
Does this turn off the heating in the night camera during the day?

File: bin/allsky_scheduler [hunk 1]
```diff
@@ -160,6 +160,7 @@ def get_device_configuration(config, section):
     d['postprocess'] = config.getboolean(section, 'postprocess')
     d['rotate180'] = config.getboolean(section, 'rotate180')
     d['overlay'] = config.getboolean(section, 'overlay')
+    d['heating'] = config.getbooean(section, 'heating')
```
Comment (mgdaily) [inline bin/allsky_scheduler]: typo - should be `getboolean`

File: pyallsky/imagecapture.py [hunk 2]
```diff
@@ -40,6 +42,11 @@ def capture_image_device(device, exposure, dark=False):
     logging.info('Connecting to camera')
     cam = AllSkyCamera(device)
 
+    if heating:
```
Comment (mgdaily) [inline pyallsky/imagecapture.py]: like you did above, could do this with the ternary operator

`cam.activate_heater() if heating else cam.deactivate_heater()`

**Review by mgdaily — APPROVED** [review]
Looks good, just a couple small things

**Comment by sfoale** [comment]
> Does this turn off the heating in the night camera during the day?

Good point. I've revised it so that the configuration controls the current camera's state but the other one is always off.

I think that's ok?

**Review by markBowman — APPROVED** [review]
lgtm

**Pull request merged**